src/gql/provider.tsx

```diff
@@ -51,7 +56,7 @@ const Provider: React.FC = ({ children }) => {
 
   const client = new ApolloClient({
     uri: endpoint,
-    link: ApolloLink.from([errorLink, authLink, uploadLink]),
+    link: ApolloLink.from([sentryLink, errorLink, authLink, uploadLink]),
```

Comment:
```suggestion
    link: ApolloLink.from([errorLink, sentryLink, authLink, uploadLink]),
```

https://github.com/DiederikvandenB/apollo-link-sentry#faq

> Note that this package (currently) only adds breadcrumbs. This means that you are still responsible for reporting errors to Sentry.

So it is desirable to actually send the error to Sentry only after it has passed through the SentryLink process.

Process flow:

```
errorLink -> sentryLink -> authLink -> uploadLink -> (sending query and error happen) -> uploadLink -> authLink -> sentryLink (attach breadcrumbs) -> errorLink
```

Reply:
Thank you!